### PR TITLE
fix: wording submit verification submitted

### PR DESCRIPTION
**Pull request by azmimuhammad**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-2486

### Changelog/Description 🗒️
- update method in laseventData listener

### WARNING ⚠️
- 

### Screenshoot/Video (Optional) 🖼️
Screenshot_9

### CODE REVIEW

File: src/views/Dashboard/Lab/NavigationLayout.vue [hunk 1]
```diff
@@ -160,7 +160,7 @@ export default {
 
     lastEventData(val) {
       if (val !== null) {
-        if (val.method === "LabRegistered") return
+        if (val.method === "LabStakeSuccessful") return
```
Comment (hilyds) [inline src/views/Dashboard/Lab/NavigationLayout.vue:163]: customer should get notification not based on `LabStakeSuccessful` event, but after trigger send email verification

**Review by hilyds — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/NavigationLayout.vue (hunk 1 above)
Comment (azmimuhammad) [reply src/views/Dashboard/Lab/NavigationLayout.vue:163]: notification already appears when sending email, you can see from the origin branch in this PR

https://github.com/azmimuhammad/debio-frontend/blob/8ac5dc1215a7ab80592c5f1e92fc2c876a9862a3/src/views/Dashboard/Lab/Registration/Services/index.vue#L465

File: src/views/Dashboard/Lab/index.vue [hunk 2]
```diff
@@ -66,11 +66,22 @@ export default {
...
     OrderList
   },
+  
+  watch: {
+    async lastEventData(val) {
+      if (val !== null) {
+        if (val.method === "LabStakeSuccessful") return
+
+        await this.$store.dispatch("substrate/getLabAccount")
+      }
```
Comment (bukandicki) [inline src/views/Dashboard/Lab/index.vue:76]: please make it simpler

**Review by bukandicki — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/index.vue (hunk 2 above)
Comment (azmimuhammad) [reply src/views/Dashboard/Lab/index.vue]: updated

**Review by bukandicki — APPROVED** [review]
Nice One

**Review by hilyds — APPROVED** [review]

**Pull request merged**